provider/win_driver_installer_test.py

```diff
@@ -197,3 +200,41 @@ def check_gagent_version(session, test, gagent_pkg_info_cmd,
     if actual_gagent_version != expected_gagent_version:
         test.fail("gagent version is not right, expected is %s but got %s"
                   % (expected_gagent_version, actual_gagent_version))
+
+
+@error_context.context_aware
+def win_installer_version_test(session, test, params):
+    """
+    Windows installer version check.
+
+    :param session: The guest session object.
+    :param test: kvm test object
+    :param params: the dict used for parameters.
+    """
+    error_context.context("Check virtio-win-installer version.",
+                          LOG_JOB.info)
+    rpm_pkg_name = process.system_output(params["rpm_install_chk_cmd"], shell=True).strip().decode()
+    if "not installed" in rpm_pkg_name:
```

Comment:
This might not be solid, `process.getstatusoutput()` and check its return value would be better.